src/SiteRecoveryManagement/Properties/AssemblyInfo.cs

```diff
@@ -20,7 +20,7 @@
 [assembly: AssemblyDescription("")]
 
 [assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.5.1.0")]
+[assembly: AssemblyFileVersion("0.6.0.0")]
```

Comment:
Do you really need to change all these assembly versions? Authentication, Compute, and SiteRecovery?